Makefile

```diff
@@ -60,12 +63,20 @@ check:
 	make -C $(wasms) check && \
```

Comment:
on line 62 (I cannot comment on it), we should also add `--features ABCI` to the `cargo check`